tests/amd/amd.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, CKEDITOR */
```

Comment:
We're gonna have describe, it and expect in every test, maybe we can "globalize" them in a single place, like `.jshintrc` file in the `tests/` directory?